toolio_agent/agent/prompts.py

```
@@ -0,0 +1,118 @@
+Prompt templates for the Agent system.
+
+This module contains all system prompt templates used by the agent,
+including response format instructions and schema definitions.
+"""
+
+import json
+
+from .models import AgentFinalResponse, AgentPlan, AgentStep
+
+
+def build_system_prompt(custom_instructions: str = None) -> str:
```

Ruff (0.14.8)

PEP 484 prohibits implicit `Optional`

Convert to `Optional[T]`

(RUF013)